cypress/support/pages/article.pageObject.js

```diff
@@ -0,0 +1,101 @@
+import PageObject from '../PageObject';
+
+class ArticlePageObject extends PageObject {
+  url = '/#/editor';
+
+  get artilcleTitleField() {
+    return cy.getByDataQa('article-title-field');
+  }
+
+  get whatIsThisArticleAboutField() {
+    return cy.getByDataQa('what-is-this-article-about');
+  }
+
+  get writeYourArticleField() {
+    return cy.getByDataQa('write-your-article-field');
+  }
+
+  get enterTagsField() {
+    return cy.getByDataQa('enter-tags-field');
+  }
+
+  get publishArticleBtn() {
+    return cy.getByDataQa('publish-article-btn');
+  }
+
+  get editArticleBtn() {
+    return cy.getByDataQa('edit-article-btn');
+  }
+
+  get deleteArticleBtn() {
+    return cy.getByDataQa('delete-article-btn');
+  }
+
+  get createdArticleTitle() {
+    return cy.getByDataQa('created-article-title');
+  }
+
+  typeArticleTitle(articleTitle) {
+    this.artilcleTitleField
+      .type(articleTitle);
+  }
+
+  updateArticleTitle(articleTitle) {
+    this.artilcleTitleField
+      .clear()
+      .type(articleTitle);
+  }
```

Comment:
```suggestion
  typeArticleTitle(articleTitle) {
    this.artilcleTitleField
      .type(articleTitle);
  }

  updateArticleTitle(articleTitle) {
    this.artilcleTitleField
      .clear()
      .type(articleTitle);
  }
```

you could leave 1 method with update article title and use it only